src/components/crowdsale/index.js

```diff
@@ -17,7 +17,7 @@ import { NAVIGATION_STEPS } from '../../utils/constants'
 import { Loader } from '../Common/Loader'
 import { CrowdsaleConfig } from '../Common/config'
 import { inject, observer } from 'mobx-react'
-import { invalidCrowdsaleExecIDAlert, invalidCrowdsaleAddrAlert, invalidNetworkIDAlert } from '../../utils/alerts'
+import { invalidCrowdsaleExecIDAlert, invalidNetworkIDAlert } from '../../utils/alerts'
```

Comment:
While `invalidNetworkIDAlert ` is not used somewhere in the code yet, but it should (related issue is https://github.com/poanetwork/token-wizard/issues/1044). I guess, we could leave it this line as is in this PR.

Reply:
Ok, It will remain that way